README.md

```diff
@@ -75,12 +75,16 @@ for boot device and rootfs device respectively. In order to run this demo, the f
 
    - press **NEXT** button:  
       ![Step 6](doc/pictures/install_step_6.png)
-   - confirm flashing SD card device by pressing **YES** on the next dialog box:  
+   - Choose **EDIT SETTINGS** if you wish to set a username and password for logging into the device. **Attention: Username and password are required otherwise you will not be able to log into the Raspberry Pi.**:  
```

Comment:
120 symbols max len move attention to next line: format as following:

```
**ATTENTION:** username and password are required otherwise, you will not be able to log into the Raspberry Pi:
```